app/workers/update_schedule_worker.rb

```diff
@@ -0,0 +1,10 @@
+class UpdateScheduleWorker
+  include Sidekiq::Worker
+
+  def self.perform
+    Student.all.each do |student|
+      student.update(at_school: false)
+      UpdateScheduleJob.perform(student)
```

Comment:
Yup. Assuming that more students are usually present than absent, then you'll have to manually adjust fewer people.